tests/unit/nupic/algorithms/anomaly_likelihood_test.py

```diff
@@ -231,10 +231,10 @@ def testEstimateAnomalyLikelihoodsMalformedRecords(self):
 
     # Generate an estimate using fake distribution of anomaly scores.
     data1 = _generateSampleData(mean=0.2)
-    data1 = data1 + [(2, 2), (2, 2, 2, 2), (), (2)]  # Malformed records
+    data1 = data1[0:1000] + [(2, 2), (2, 2, 2, 2), (), (2)]  # Malformed records
 
     likelihoods, avgRecordList, estimatorParams = (
-      an.estimateAnomalyLikelihoods(data1[0:1000])
+      an.estimateAnomalyLikelihoods(data1[0:1004])
     )
     self.assertEqual(len(likelihoods), 1000)
```

Comment:
The 4 malformed inputs get culled out. leaving 1000 well-formed records
right?

I put in 1000 good ones and 4 bad ones, making it 1004 records total.
The code bypasses the bad ones returning 1000 likelihoods - which then the
assertion checks for...

correct?

On Mon, Feb 23, 2015 at 4:46 PM, breznak <notifications@github.com> wrote:

> In tests/unit/nupic/algorithms/anomaly_likelihood_test.py
> <https://github.com/numenta/nupic/pull/1855#discussion_r25210954>:
>
> >
> >      likelihoods, avgRecordList, estimatorParams = (
> > -      an.estimateAnomalyLikelihoods(data1[0:1000])
> > +      an.estimateAnomalyLikelihoods(data1[0:1004])
> >      )
> >      self.assertEqual(len(likelihoods), 1000)
>
> here it's 1004
> https://github.com/numenta/nupic/pull/1855/files#diff-938a3b6b5c718f261186df565988e7a4R237
> while there is 1000
> https://github.com/numenta/nupic/pull/1855/files#diff-938a3b6b5c718f261186df565988e7a4R240
>
> or do I miss something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numenta/nupic/pull/1855/files#r25210954>.
>



-- 
*We find it hard to hear what another is saying because of how loudly "who
one is", speaks...*